eslint.config.js

```diff
@@ -0,0 +1,444 @@
+const { FlatCompat } = require("@eslint/eslintrc");
+const pluginReact = require("eslint-plugin-react");
+const pluginImport = require("eslint-plugin-import");
+const pluginNode = require("eslint-plugin-node");
+const pluginTypeScript = require("@typescript-eslint/eslint-plugin");
+
+const compat = new FlatCompat({
```

Comment:
As the warning is howing here (in the [files tab of the PR](https://github.com/metabrainz/bookbrainz-site/pull/1091/files) on github), this compat configuration you create here is never used.
Either something is wrong with the configuration as it stands, or the FlatCompat and associated package are not actually needed...
Not sure which it is.